bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitChangeListener.java

```diff
@@ -88,31 +89,40 @@
         metadataChangeListener = new RegistryChangeListener<Metadata>() {
             @Override
             public void added(final Metadata metadata) {
-                try {
-                    markDirty(itemRegistry.getItem(metadata.getUID().getItemName()));
-                } catch (ItemNotFoundException e) {
-                    logger.debug("Could not found item for metadata {}", metadata);
+                final MetadataKey uid = metadata.getUID();
+                if (uid.getNamespace().equalsIgnoreCase(HomekitAccessoryFactory.METADATA_KEY)) {
+                    try {
+                        markDirty(itemRegistry.getItem(uid.getItemName()));
+                    } catch (ItemNotFoundException e) {
+                        logger.debug("Could not found item for metadata {}", metadata);
```

Comment:
Could not find... 

Please fix everywhere.

Reply:
done